### PR TITLE
Update major changes in datamodel and finish of get request endpoint

### PR DESCRIPTION
**Pull request by samBAM089**
after realizing how the real data flow will be in the final application, I had to make some major changes.
mainly removed all Dbs and added a new one just for users.

methods and tests were all green.

### CODE REVIEW

**Review by dride-for-sure — CHANGES_REQUESTED** [review]

File: backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import java.util.List;
+
+@Repository
+public class UserDb {
+
+    private List<User> userList;
+
+    public UserDb() {
+        this.userList = new ArrayList<>();
+        this.userList.add(User.builder()
```
[the remark this answers is not in the document]
Comment (samBAM089) [reply backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java:17]: nice, i was wondering, why it did not take "List.of()" as Argument ^^

File: backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        this.userList.add(userToAdd);
+    }
+
+    public User getLogInUser() {
+        return this.userList.get(0);
+    }
+
+    public void clear() {
+        this.userList.clear();
+    }
```
[the remark this answers is not in the document]
Comment (samBAM089) [reply backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java:40]: that's for testing purposes ^^'

File: backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                .build());
+    }
+
+    public void addUser(User userToAdd) {
+        this.userList.add(userToAdd);
+    }
+
+    public User getLogInUser() {
+        return this.userList.get(0);
+    }
```
[the remark this answers is not in the document]
Comment (samBAM089) [reply backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java:36]: well, if it was an native app, yes ^^'

File: backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (dride-for-sure) [reply backend/src/main/java/de/sambam/gamenighttracker/db/UserDb.java:17]: List.of() is possible true :)

**Review by dride-for-sure — APPROVED** [review]
So that your software also runs stable, I would swap maumau for dame! 🚀

**Pull request merged**